### PR TITLE
better video handling for attached media

### PR DESCRIPTION
**Pull request by Devenwel**
@DorienD can you take a look please, play arrow styling is wrong and sometimes I am not getting a thumbnail when I embed via URL

### CODE REVIEW

File: modules/mod_ginger_base/templates/attached-media/attached-media-thumb.video.tpl [hunk 1]
```diff
@@ -1,39 +1,72 @@
+{% if m.rdf[id.rdf] as rdf %}{% endif %}
```
Comment (ddeboer) [inline modules/mod_ginger_base/templates/attached-media/attached-media-thumb.video.tpl]: Can be removed.

Comment (DorienD) [inline modules/mod_ginger_base/templates/attached-media/attached-media-thumb.video.tpl]: Removed it

**Comment by DorienD** [comment]
I have removed the empty rdf check and added some styling for the play icon in the css instead of the icon element.

**Comment by ddeboer** [comment]
Is this ready to be merged?

**Comment by DorienD** [comment]
@Devenwel This was done right?

**Pull request closed without merging**